backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -14,6 +14,9 @@ body {
 .admin-nav {
   @include position(absolute, 0 null 0 0);
   width: $width-sidebar;
+  img {
+    max-width: 100%;
+  }
 }
```

Comment:
Indeed rightly pointed. Thanks for the reply! It's also great that you have established a style.